molecule/__init__.py

```diff
@@ -18,8 +18,20 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
+import signal
+import sys
+
 from pbr import version
 
+
+def signal_handler(signal, frame):
+    msg = '{red}Exiting molecule on Ctrl+c.{reset}'
+    print(msg.format(red='\033[31m', reset='\033[0m'))
```

Comment:
Even in light of my comment about it making trapping ^C significantly less effective?